civiwiki/civiwiki/urls.py

```diff
@@ -25,7 +25,8 @@
     url(r'^api/', include(urls_write)),
     url(r'^api/', include(urls_read)),
     url(r'^api/', include(urls_auth)),
-    url(r'^',include(urls_templates)),
+    url(r'^/',include(urls_templates)),
+    url(r'^$',include(urls_templates)),
```

Comment:
So at first the way we had it was like a base case where that matched everything that didn't come before it. I want it to specifically match the empty string or a path that is in url templates. This way urls like /media and /static aren't ignored